### ISSUE TITLE
Build fails (Arch Linux)

### ISSUE THREAD

**Issue by keidax** [redox-os/redox#582]
**Reproduction**: [describe how you are able to reproduce ("trigger") this bug/issue.]

I followed the manual setup guide and tried to build Redox, but I get the following error:
```
$ make all
RUST_BACKTRACE=1 rustc --target=i386-unknown-redox.json -L build/i386-unknown-redox/debug -C no-prepopulate-passes -C no-stack-check -C opt-level=2 -Z no-landing-pads -A dead_code -o build/i386-unknown-redox/debug/libsystem.rlib crates/system/lib.rs
crates/system/externs.rs:2:1: 2:35 error: definition of an unknown language item: `stack_exhausted`. [E0522]
crates/system/externs.rs:2 extern "C" fn stack_exhausted() {}
                           ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
crates/system/externs.rs:2:1: 2:35 help: run `rustc --explain E0522` to see a detailed explanation
error: aborting due to previous error
Makefile:443: recipe for target 'build/i386-unknown-redox/debug/libsystem.rlib' failed
make: *** [build/i386-unknown-redox/debug/libsystem.rlib] Error 101
```
I also tried [these instructions](https://github.com/redox-os/redox#-help-redox-wont-compile-) with no luck.

**Build information**: [output of `rustc -V`, `git rev-parse HEAD`, `qemu-i386 -version`, `uname -a`, etc.]
```
$ rustc -V
rustc 1.9.0-nightly (0dcc413e4 2016-03-22)
$ git rev-parse HEAD
cb17447b
$ uname -a
Linux arch 4.4.5-1-ARCH #1 SMP PREEMPT Thu Mar 10 07:38:19 CET 2016 x86_64 GNU/Linux

**Comment by nounoursheureux** [comment]
Fixed by d8f0d5d145485b3fb2b1ae9d42a9375997431882

**Comment by keidax** [comment]
That was quick!
